Library/UIButton+HapticFeedback.swift

```diff
@@ -0,0 +1,14 @@
+import UIKit
+
+extension UIButton {
+
+  @available(iOS 10.0, *)
+  public func generateSelectionFeedback() {
+    UISelectionFeedbackGenerator().selectionChanged()
```

Comment:
This should be moved to `UIFeedbackGenerator` extension and called like `UIFeedbackGenerator(). ksr_selectionFeedback()`